lib/xnvme_be_cbi_async_posix.c

```diff
@@ -11,7 +11,6 @@
 #include <inttypes.h>
 #include <errno.h>
 #include <aio.h>
-#include <libxnvme_spec_fs.h>
```

Comment:
This should include libxnvme.h

Reply:
Since the internal headers still include other headers, then it gets ``libxnvme.h`` via ``xnvme_be.h``